buildconfig/makeref.py

```
@@ -27,13 +28,41 @@ def run():
         if full_generation_flag:
             subprocess_args.append('-E')
         print("Executing sphinx in subprocess with args:", subprocess_args)
-        return subprocess.run(subprocess_args).returncode
+        returncode = subprocess.run(subprocess_args).returncode
+        if returncode == 0:
+            add_js_files()
+        return returncode
     except Exception:
         print('---')
         print('Have you installed sphinx?')
         print('---')
         raise
 
+def add_js_files():
+    files_to_change = ['search.html', 'genindex.html']
+    tag_to_find = '<div class="body" role="main">'
+    tag_to_add = f'<script src="script.html"></script>'
+    try:
+        for file in files_to_change:
+            with open(os.path.join(rst_build_dir, file),'r') as f:
+                data = f.read()
+                index = data.find(tag_to_find)
+                if index == -1:
+                    return Exception
+                index += len(tag_to_find) - 1
+                data = data[:index + 1] + '\n' + tag_to_add + data[index:]
+            with open(os.path.join(rst_build_dir, file),'w') as f:
+                f.write(data)
+        original = r"{}".format(os.path.join(rst_source_dir, 'themes/classic') + '\script.html')
+        target = r"{}".format(f'{rst_build_dir}\script.html')
+        shutil.copyfile(original, target)
+    except Exception:
+        print('---')
+        print('Failed to add js files')
+        print('---')
+        print(Exception)
+        return Exception
```

Few issues: 
1. Why do you print ``Exception`` class into the console. Maybe you want to re-raise it again?
2. ``return Exception`` Again, why?
3. This part probably was intended to be for debugging reasons, but instead it hides the error.

---

Looks resolved?